src/components/page-details.jsx

```diff
@@ -176,15 +170,34 @@ export default class PageDetails extends React.Component {
 
   _versionsToRender () {
     const [fromId, toId] = (this.props.match.params.change || '').split('..');
-    let from = this.state.page.versions.find(v => v.uuid === fromId);
-    let to = this.state.page.versions.find(v => v.uuid === toId);
+    const versions = this.state.page.versions;
```

Comment:
Now that we have the concept, I think the right default change is `..$`, so:

```js
const [fromId, toId] = (this.props.match.params.change || '..$').split('..');
```

That might help reduce complexity slightly.